packages/backend/src/core/queue/TaskQueue.ts

```diff
@@ -202,15 +211,13 @@ export class TaskQueue<T> {
     }
   }
 
-  // WARNING: this method clears the queue, be cautious when using it
-  // some Updaters will not function properly after you unhalt them using this method
-  // because they rely on the start() function which is called only once
-  // so use it only in Updaters with generic (updating all the missing data) update() function
-  // or rewrite the logic of your updater
-  unhaltIfNeeded() {
-    if (this.halted) {
-      this.queue.splice(0, this.queue.length)
-      this.halted = false
-    }
+  // WARNING: this method should be used only in tests
+  stop() {
+    this.stopped = true
+  }
+
+  // WARNING: this method should be used only in tests
+  clear() {
+    this.queue.splice(0, this.queue.length)
```

Comment:
Use `/**` for doc comments. This comment won't show up on hover. Also you could make it obvious from the method name e.g. `_TEST_ONLY_stop()` and `_TEST_ONLY_clear()`